### PR TITLE
8342911: Remove calls to doPrivileged in controls

### PR DESCRIPTION
**Pull request by andy-goryachev-oracle**
Removes doPrivileged in the following module:

- javafx.controls

See JDK-8342441 for details.

As a helpful hint for reviewers, I recommend reviewing this using the "Hide whitespace" option.

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed (2 reviews required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer), 1 [Author](https://openjdk.org/bylaws#author))

### Issue
 * [JDK-8342911](https://bugs.openjdk.org/browse/JDK-8342911): Remove calls to doPrivileged in controls (**Bug** - P3)

### Reviewers
 * [Kevin Rushforth](https://openjdk.org/census#kcr) (@kevinrushforth - **Reviewer**)
 * [Ajit Ghaisas](https://openjdk.org/census#aghaisas) (@aghaisas - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jfx.git pull/1615/head:pull/1615` \
`$ git checkout pull/1615`

Update a local copy of the PR: \
`$ git checkout pull/1615` \
`$ git pull https://git.openjdk.org/jfx.git pull/1615/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 1615`

View PR using the GUI difftool: \
`$ git pr show -t 1615`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jfx/pull/1615.diff">https://git.openjdk.org/jfx/pull/1615.diff</a>

</details>
<details><summary>Using Webrev</summary>

[Link to Webrev Comment](https://git.openjdk.org/jfx/pull/1615#issuecomment-2442041491)
</details>

### CODE REVIEW

**Review by kevinrushforth — APPROVED** [review]
This looks good to me. I left a minor comment about the "TODO" you added (doesn't seem related or needed). I'll reapprove if you decide to change it.

**Comment by kevinrushforth** [comment]
A second pair of eyes would be good.

@arapte or @jayathirthrao Can one of you review?

/reviewers 2

File: modules/javafx.controls/src/main/java/javafx/scene/control/skin/MenuBarSkin.java [hunk 1]
```diff
@@ -108,10 +105,7 @@ public class MenuBarSkin extends SkinBase<MenuBar> {
     static {
         final Predicate<Window> findStage = (w) -> w instanceof Stage;
         @SuppressWarnings("removal")
```
Comment (kevinrushforth) [inline modules/javafx.controls/src/main/java/javafx/scene/control/skin/MenuBarSkin.java:107]: I missed this: You can remove this now-unneeded SuppressWarnings.

**Review by kevinrushforth — APPROVED** [review]

**Review by aghaisas — APPROVED** [review]
Looks good to me.

**Pull request closed without merging**